src/Sarif.Converters.UnitTests/PylintConverterTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I got to do git push by force to push new changes as few changes are mainlined after old commits.